ibis/backends/chdb/tests/conftest.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+import os
+from typing import TYPE_CHECKING, Any, Callable
+
+import pytest
+
+import ibis
+import ibis.expr.types as ir
+from ibis.backends.tests.base import BackendTest
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+    from pathlib import Path
+
+
+IBIS_TEST_CHDB_DB = os.environ.get("IBIS_TEST_DATA_DB", "ibis_testing")
+
+
+class TestConf(BackendTest):
+    check_dtype = False
+    supports_window_operations = False
+    returned_timestamp_unit = "s"
+    supported_to_timestamp_units = {"s"}
+    supports_floating_modulus = False
+    supports_json = False
+    force_sort = True
+    rounding_method = "half_to_even"
+    deps = ("chdb",)
+
+    @property
+    def native_bool(self) -> bool:
+        return True
+        # [(value,)] = self.connection.con.query("SELECT true").result_set
+        # return isinstance(value, bool)
+
+    @property
+    def test_files(self) -> Iterable[Path]:
+        return self.data_dir.joinpath("parquet").glob("*.parquet")
+
+    @property
+    def ddl_script(self) -> Iterable[str]:
+        parquet_dir = self.data_dir / "parquet"
+        for sql in super().ddl_script:
+            yield sql.format(parquet_dir=parquet_dir)
```

Comment:
I'd rather not introduce templating if we can avoid it. How about hard coding the path to `ci/ibis-testing-data/...` like we do with everything else?